### PR TITLE
CloudWatch: remove references to json in template variable docs

### LINKED ISSUE #48007: CloudWatch: Document new Variable Editor
Now that the variable editor has changed, it needs new documentation.

### PR DESCRIPTION
**Pull request by iwysiu**
**What this PR does / why we need it**:
The new template variable form doesn't require json formatted filters, so this removes the references to them.

Screen Shot 2022-05-02 at 12 11 45 PM

**Which issue(s) this PR fixes**:

Fixes #48007

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ For an introduction to templating and template variables, refer to the [Templati
 
 ## Query variable
 
-The CloudWatch data source provides the following queries that you can specify in the `Query Type` field in the Variable edit view. They enable you to fill a variable's options list with values such as `region`, `namespaces`, `metric names` and `dimension keys/values`.
+The CloudWatch data source provides the following queries that you can specify in the `Query Type` field in the Variable edit view. They enable you to fill a variable's options list with values such as `regions`, `namespaces`, `metric names` and `dimension keys/values`.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:16]:
```suggestion
You can use the following CloudWatch data source queries to specify the `Query Type` field in the Variable edit view. Use them to fill a variables options list with values like `regions`, `namespaces`, `metric names`, and `dimension keys/values`.
```

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 2]
```diff
@@ -35,29 +35,12 @@ For details about the metrics CloudWatch provides, please refer to the [CloudWat
...
-{"Environment":${env:json}}
-```
-
 ## ec2_instance_attribute examples
 
-### JSON filters
+### Filters
 
-The `ec2_instance_attribute` query takes `filters` in JSON format.
+The `ec2_instance_attribute` query takes in `filters`, each as a filter name and a comma-separated list of values.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:42]:
```suggestion
The `ec2_instance_attribute` query takes in `filters`as a filter name and a comma-separated list of values.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md [hunk 3]
```diff
@@ -35,29 +35,12 @@ For details about the metrics CloudWatch provides, please refer to the [CloudWat
...
-
 ## ec2_instance_attribute examples
 
-### JSON filters
+### Filters
 
-The `ec2_instance_attribute` query takes `filters` in JSON format.
+The `ec2_instance_attribute` query takes in `filters` as a filter name and a comma-separated list of values.
 You can specify [pre-defined filters of ec2:DescribeInstances](http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html).
-Note that the actual filtering takes place on Amazon's servers, not in Grafana.
```
Comment (kevinwcyu) [inline docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:54]: Is this line still relevant or was it only specific to the JSON format?

**Review by kevinwcyu — APPROVED** [review]
LGTM, just one question.

**Review by achatterjee-grafana — APPROVED** [review]

File: docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md (hunk 3 above)
Comment (iwysiu) [reply docs/sources/datasources/aws-cloudwatch/template-queries-cloudwatch.md:54]: Still relevant! Those are the filter names on aws's end.

**Pull request merged**